chef_master/source/_templates/nav-docs.html

```diff
@@ -2211,84 +2177,21 @@
                 "hasSubItems": false,
                 "url": "/fips.html"
```

Comment:
FIPS should really get a new home now that there's no automate stuff in it. It's an infra doc now.